File: Java im portant question/CaseType.java

/*
Write a program to tell that the first letter is uppercase or lowercase.
 */
import java.util.Scanner;
public class CaseType
{
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter word: ");
        char word = sc.next().charAt(0);

        if (word >= 'A' && word <= 'Z'){
            System.out.println("Upper case");
        }
        else {
            System.out.println("Lower case");
        }
    }
}

File: Js important question/CaseType.js
// Write a program to tell that the first letter is uppercase or lowercase.
Word = prompt("Enter the word!! ")
word2 = String(Word).charAt(0).trim();

if(word2 >= 'A' && word2 <= 'Z'){
    console.log("Upper Case");
}
else{
    console.log("Lower Case");
}